packages/ui-dom/src/Dropdown/Option.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import styled from 'styled-components'
+import {option} from '@emcasa/ui/lib/components/Dropdown'
+
+const DropdownOption = styled(({children, id, className, style, onSelect}) => {
+  return (
+    <button
+      tabIndex={0}
+      id={id}
+      className={className}
+      style={style}
+      onMouseDown={onSelect}
```

Comment:
Selecting an option should automatically close the dropdown. When I tested, I had to click outside for the options to close.